test/bats/onchain-send.bats

```diff
@@ -134,21 +135,33 @@ teardown() {
   [[ "${settled_status}" = "SUCCESS" ]] || exit 1
 }
 
-@test "onchain-send: settle intraledger" {
+@test "onchain-send: settle intraledger, with no contacts check" {
   alice_token_name="$ALICE_TOKEN_NAME"
   alice_btc_wallet_name="$alice_token_name.btc_wallet_id"
   alice_usd_wallet_name="$alice_token_name.usd_wallet_id"
 
-  bob_token_name="$BOB_TOKEN_NAME"
-  bob_btc_wallet_name="$bob_token_name.btc_wallet_id"
+  recipient_token_name="user_$RANDOM"
+  recipient_phone="$(random_phone)"
+  login_user \
+    "$recipient_token_name" \
+    "$recipient_phone"  \
+    "$CODE"
+  user_update_username "$recipient_token_name"
+  btc_recipient_wallet_name="$recipient_token_name.btc_wallet_id"
+
+  # Check is not contact before send
+  run is_contact "$alice_token_name" "$recipient_token_name"
```

Comment:
not a blocker but should we rename "alice" prefix ?

Reply:
Hm, `alice` is a fixed user but the `recipient` I renamed is a transient one created for the test. Maybe rename would make sense if we made Alice random too, which we probably need right now here